arch/cortex-m/src/scb.rs

```diff
@@ -300,8 +300,7 @@ pub unsafe fn disable_fpca() {
     SCB.cpacr
         .modify(CoprocessorAccessControl::CP10::CLEAR + CoprocessorAccessControl::CP11::CLEAR);
 
-    llvm_asm!("dsb");
-    llvm_asm!("isb");
+    asm!("dsb", "isb");
```

Comment:
should this also have `options(nomem, nostack, preserves_flags)`? It seems that these barrier instructions should not be touching memory or stack or condition codes

Reply:
Thank you for your feedback :+1: I added the options to the `asm!` invocation :)